### DISCUSSION TITLE
Thunder client of vs code not passing authentication details from the parent request to child request

### DISCUSSION THREAD

**Discussion by govind-mishra** [thunderclient/thunder-client-support#1335] [Q&A] [answered]
As we know like Postman , In ThunderClient, we can set the Authorization header as "Inherit Authentication values from Parent" for a request.
I am passing jwtToken value in header of my login request and wanna delete that token on logout request. But  I am having trouble passing the JWT token from the login request to the logout request, even though I've enabled "Inherit Authentication values from Parent."

**Comment by rangav** [comment]
You should use `Auth` tab to use inherit Parent. But your are using headers tab now

**Reply by govind-mishra** [reply]
Yeah I am using auth tab to use inherit parent. In first auth tab there is None subtab showing `Inherit Authentication values from Parent`

**Reply by rangav** [reply]
Can you share screenshot

**Comment by govind-mishra** [comment]
parent request
<img width="726" alt="image" src="https://github.com/rangav/thunder-client-support/assets/49047340/2a0953c5-3a19-4240-a620-bdee37627ddd">
child request
<img width="735" alt="image" src="https://github.com/rangav/thunder-client-support/assets/49047340/28dd422e-62e1-4d29-8133-ca9535abd04d">

**Reply by rangav** [reply]
Run the child request again

**Comment by govind-mishra** [comment]
<img width="735" alt="image" src="https://github.com/rangav/thunder-client-support/assets/49047340/741824cb-7129-40a2-9ab4-c1355f9faafb">

**Reply by rangav** [reply, marked as answer]
You should set the Auth details in collection settings or folder settings 

and see logs
https://github.com/rangav/thunder-client-support#logs

**Comment by govind-mishra** [comment]
Hi Rangav!  thanks man I got it .
